src/Controller/ArticlesController.php

```diff
@@ -978,7 +978,7 @@ private function articlePageArguments(Request $request, string $id, int $version
                 $infoBars = [];
                 if (false === $isMagazine) {
                     $infoBarText = sprintf(
-                        'eLife\'s peer-review process is changing. From early next year, we will no longer make accept/reject decisions after peer review. <a href="%s" class="">About the new process.</a>',
+                        'eLife\'s peer-review process is changing. From January 2023, we will no longer make accept/reject decisions after peer review. <a href="%s" class="">About the new process.</a>',
```

Comment:
Should read: "In the new process" rather than "From January 2023"

Reply:
Have made changes.